src/v/redpanda/application.cc

```diff
@@ -64,6 +65,29 @@ application::application(ss::sstring logger_name)
 
   };
 
+static void log_system_resources(
+  ss::logger& log, const boost::program_options::variables_map& cfg) {
+    /**
+     * IMPORTANT: copied out of seastar `resources.cc`, if it is going to change
+     * we have to change logic in here
```

Comment:
someone that reads this in the future is not going to know what here in this code is sensitive to changes in seastar. should we create a ticket to expose this information from seastar directly, instead?

Reply:
is the sensitive part the 1536 * 0.7? if so we can make that a constant no?

Reply:
I like the idea of exposing this info out of `reactor.cc`

Reply:
I will create a ticket, it isn't super easy to expose it from seastar, reserved memory is only used during initialization phase and it is not exposed anywhere.